rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -594,6 +570,7 @@ type ExtendibleProvision extends BuyerSeller: <"A data defining:  an option to e
     noticeDeadlineDateTime zonedDateTime (0..1) <"A specific date and time for the notice deadline">
     extensionTerm RelativeDateOffset (0..1) <"The length of each extension period relative to the effective date of the preceding contract.">
     extensionPeriod AdjustableRelativeOrPeriodicDates (0..1) <"The period within which notice can be given that the contract will be extended.">
+    exerciseTerms ExerciseTerms (1..1)
```

Comment:
Please a brief comment for ```exerciseTerms```, like you have done for the instance under ```OptionPayout```

Reply:
Done - in #2687